<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicUserRecordWidget</class>
 <widget class="QWidget" name="MusicUserRecordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户设置 </string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>530</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>用户设置 </string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>530</width>
     <height>435</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,200)</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>417</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
    border:none;
}
QTabBar::tab {
	height:30px;
	width:125px;
    background-color: #BFBFBF;
    color: white;
}
QTabBar::tab:selected {
    background-color: #80B7F1;
    color: white
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">
	color:#666666;</string>
     </property>
     <attribute name="title">
      <string>个人资料</string>
     </attribute>
     <widget class="QLabel" name="label_F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>积分:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="userTimeLabel_F">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="userIDLabel_F">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>性别:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>生日:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>城市:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7F">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>签名:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmButton_F">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nicknameEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="maleRadioButton_F">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>153</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>男</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="femaleRadioButton_F">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>153</y>
        <width>60</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>女</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="birthDateEdit_F">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>190</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cityComboBox_F">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="countryComboBox_F">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="signatureEdit_F">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">color:#666666;</string>
     </property>
     <attribute name="title">
      <string>修改头像</string>
     </attribute>
     <widget class="QLabel" name="label_S">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight:bold</string>
      </property>
      <property name="text">
       <string>设置您喜爱的一个头像:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2S">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>45</y>
        <width>461</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>(请选择图片文件，最佳尺寸400X400，支持JPG,JPEG,PNG,GIF)</string>
      </property>
     </widget>
     <widget class="QLabel" name="bigPixmapLabel_S">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="smlPixmapLabel_S">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5S">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>130</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>头像预览</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFileButton_S">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>选择头像</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>修改密码</string>
     </attribute>
     <widget class="QLabel" name="label_T">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=&quot;#FF0000&quot;&gt;*&lt;/font&gt;原密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="MusicUserLineEdit" name="originPwdEdit_T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2T">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=&quot;#FF0000&quot;&gt;*&lt;/font&gt;新密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="MusicUserLineEdit" name="newPwdEdit_T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3T">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=&quot;#FF0000&quot;&gt;*&lt;/font&gt;确认密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="MusicUserLineEdit" name="newCPwdEdit_T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="verificationCodeEdit_T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>215</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4T">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=&quot;#FF0000&quot;&gt;*&lt;/font&gt;验证码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmButton_T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>285</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5T">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>72</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密码强度：弱</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6T">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>140</y>
        <width>15</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>强</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pwdStrengthT1">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>145</y>
        <width>35</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#BFBFBF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pwdStrengthT2">
      <property name="geometry">
       <rect>
        <x>227</x>
        <y>145</y>
        <width>35</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#BFBFBF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pwdStrengthT3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>145</y>
        <width>35</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#BFBFBF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelRighT1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>53</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image:url(:/tiny/lb_right)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelRighT2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>103</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image:url(:/tiny/lb_right)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelRighT3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>173</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image:url(:/tiny/lb_right)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MusicCodeArea" name="verificationCode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>215</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <property name="styleSheet">
      <string notr="true">
	color:#666666;</string>
     </property>
     <attribute name="title">
      <string>账号安全</string>
     </attribute>
     <widget class="QLabel" name="label_FF1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>415</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font size=5&gt;&lt;b&gt;安全邮箱&lt;/b&gt;&lt;/font&gt;（安全邮箱可用于找回密码，验证后还可以作为登录邮箱）</string>
      </property>
     </widget>
     <widget class="Line" name="line_FF">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>115</y>
        <width>420</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_FF2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>225</y>
        <width>420</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>415</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font size=5&gt;&lt;b&gt;安全手机&lt;/b&gt;&lt;/font&gt;（安全手机可用于找回密码，验证后还可以作为登录手机）</string>
      </property>
     </widget>
     <widget class="Line" name="line_FF3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>335</y>
        <width>420</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>415</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;font size=5&gt;&lt;b&gt;安全问题&lt;/b&gt;&lt;/font&gt;（忘记密码时，通过安全问题重新设置密码）</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#80B7F1;font-size:15px;font-weight:bold</string>
      </property>
      <property name="text">
       <string>g******ky@163.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#80B7F1;font-size:15px;font-weight:bold</string>
      </property>
      <property name="text">
       <string>159******49</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>状态:&lt;font color=&quot;#000000&quot; size=4 &gt;&lt;b&gt;已验证&lt;/b&gt;&lt;/font&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF7">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>状态:&lt;font color=&quot;#000000&quot; size=4 &gt;&lt;b&gt;已验证&lt;/b&gt;&lt;/font&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_FF8">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>285</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>状态:&lt;font color=&quot;#000000&quot; size=4 &gt;&lt;b&gt;已设置&lt;/b&gt;&lt;/font&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicUserLineEdit</class>
   <extends>QLineEdit</extends>
   <header>musicuserlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicCodeArea</class>
   <extends>QLabel</extends>
   <header>musiccodearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
